LeaderboardBackend/Models/Entities/User.cs

```diff
@@ -4,61 +4,92 @@
 
 namespace LeaderboardBackend.Models.Entities;
 
+/// <summary>
+///     Represents a user account registered on the website.
+/// </summary>
 public class User
 {
 	/// <summary>
-	///     A GUID that identifies the User. Generated on creation.
+	///     The unique identifier of the `User`.<br/>
+	///     Generated on creation.
 	/// </summary>
-	/// <example>4b3835ca-dee1-4019-82b4-d2d26a7cce74</example>
 	public Guid Id { get; set; }
 
 	/// <summary>
-	///     The User's name. Must be:
+	///     The username of the `User`. It:
 	///     <ul>
-	///       <li>between 2 - 25 characters inclusive; and</li>
-	///       <li>an alphanumeric sequence, each separated by zero or one of:</li>
+	///       <li>must be [2, 25] in length;</li>
+	///       <li>must be made up of letters sandwiching zero or one of:</li>
```

Comment:
Should be "alphanumeric characters" instead of "letters", actually. We do allow numbers and letters.